### ISSUE TITLE
logMgr.removeTransport

### ISSUE THREAD

**Issue by eridal** [jpravetz/epdoc-logger#1]
# Test Case

```js
var epdocLogger = require('../index');

describe('logMgr.removeTransport', function () {

    var logMgr = epdocLogger.getLogManager();

    it ('should allow to remove transports', function (done) {
        // 1. add any transport
        logMgr.addTransport('console')

        // 2. try to remove any of these
        logMgr.removeTransport('console', done)
    });
});
```

# Error ouput

```txt
TypeError: transport.destroy is not a function
      at src/log_mgr.js:312:31
      at LogManager.removeTransport (src/log_mgr.js:311:27)
      at Context.<anonymous> (test/logMgr.removeTransport.js:12:16)
````

**Comment by eridal** [comment]
# Workaround

Pass the object that LogManager expects

```js
var transportHack = {
    type: 'console',
    destroy: function (next) {
        next()
    },
};
logMgr.removeTransport(transportHack);
```

**Comment by jpravetz** [comment]
Thanks for reporting. Fixed in 2.1.3.
